letsdrill.rb

```diff
@@ -1,13 +1,20 @@
-def get_letter_grade(integer)
- 
-  #Put your code here!
+def get_letter_grade(grade)
 
-end
+    return "F" if grade < 60
 
-def shortest_string(array)
+    return "D" if grade < 70
+
+    return "C" if grade < 80
 
-  #Put your code here!
+    return "B" if grade < 90
 
+    return "A"
+end
+
+def shortest_string(array)
+    return array.reduce do |shortest, current|
```

Comment:
Rubyists tend to avoid explicit `return` except for guard clauses (like your implementation of `get_letter_grade`).